app/assets/javascripts/darkswarm/services/authentication_service.js.coffee

```diff
@@ -25,9 +25,3 @@ Darkswarm.factory "AuthenticationService", (Navigation, $modal, $location, Redir
     isActive: Navigation.isActive
 
     close: ->
-      if location.pathname in ["/", "/checkout"]
```

Comment:
Ah yes, confirmed the registration page doesn't redirect. I'll take a look at it. Do you think it needs another PR? The change doesn't make much sense outside of this PR...